tasks/utils/deploy-utils.ts

```diff
@@ -25,6 +26,9 @@ export const deployContract = async <T extends Contract>(
     return contract
 }
 
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const logger = (...args: string[]) => debug(`mstable:task:${args.join(":")}`)
```

Comment:
we can move this into its own `tasks/utils/logger.ts` file.

Passing the the args and joining with `:` is a cool idea. I like